pom.xml

```diff
@@ -569,6 +569,8 @@
 					<includes>
 						<include>com.avanza.*</include>
 					</includes>
+					<groupId>com.avanza.*</groupId>
+					<artifactId>*</artifactId>
```

Comment:
This looks right 👍 
Not sure if the existing `<includes>` configuration does anything, can´t find it documented at https://www.mojohaus.org/versions-maven-plugin/set-mojo.html